### PR TITLE
bdsim: add new package

### PR DESCRIPTION
**Pull request by vvolkl**

### CODE REVIEW

File: var/spack/repos/builtin/bdsim/package.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
```
Comment (vvolkl) [inline var/spack/repos/builtin/bdsim/package.py]:
```suggestion
from spack.package import *
```

File: var/spack/repos/builtin/bdsim/package.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+       to simulate both the transport of particles in an accelerator and their 
+       interaction with the accelerator material"""
+
+    homepage = "http://www.pp.rhul.ac.uk/bdsim/manual/index.html"
+    url      = "https://bitbucket.org/jairhul/bdsim/get/v1.6.0.tar.gz"
+    git      = "https://bitbucket.org/jairhul/bdsim/src/master/"
+
+    tags = ['hep']
+
+    maintainers = ['ganis']
```
Comment (vvolkl) [inline var/spack/repos/builtin/bdsim/package.py]:
```suggestion
    maintainers = ['gganis']
```

File: var/spack/repos/builtin/bdsim/package.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Bdsim(CMakePackage):
+    """Beam Delivery Simulation (BDSIM) is a C++ program that utilises the Geant4 toolkit
+       to simulate both the transport of particles in an accelerator and their
```
Comment (vvolkl) [inline var/spack/repos/builtin/bdsim/package.py]:
```suggestion
       to simulate both the transport of particles in an accelerator and their
```

**Review by alalazo — CHANGES_REQUESTED** [review]
The package is in the wrong folder. It should be:
```console
var/spack/repos/builtin/packages//bdsim
```

**Comment by vvolkl** [comment]
Oh, good catch, my bad copying this from another recipe repo. Thanks!

**Review by alalazo — APPROVED** [review]

**Pull request merged**